flagging_site/static/README.md

```diff
@@ -0,0 +1,4 @@
+# content of static
+
+- images : contains all iamges
+- style.css : style sheet 
```

Comment:
I wouldn't include a README in `static`. As long as you know what the file is (i.e. a file for static resources that Flask treats as special), it's fairly self-explanatory everything that's in it. Also all files in `static` are accessible when the site runs, as far as I am aware.